packages/gatsby-theme-docs/src/components/overlay.js

```diff
@@ -1,8 +1,16 @@
 import styled from '@emotion/styled';
+import { keyframes } from '@emotion/react';
+import { designSystem } from '@commercetools-docs/ui-kit';
+
+const backgroundAnimation = keyframes`
+  from { background-color: rgba(0, 0, 0, 0); }
+  to { background-color: rgba(0, 0, 0, 0.5); }
+`;
 
 const Overlay = styled.div`
-  background-color: rgba(0, 0, 0, 0.5);
-  z-index: 20;
+  animation: ${backgroundAnimation} 0.3s;
```

Comment:
Nitpick: Please use the same transition curve like the menu, currently the background seems to still change while the menu is already perceived to be fully opened. 

```suggestion
  animation: ${backgroundAnimation} 0.3s ease-out;
```